vpr/src/route/route_timing.cpp

```diff
@@ -1982,3 +1995,26 @@ static void init_net_delay_from_lookahead(const RouterLookahead& router_lookahea
         }
     }
 }
+
+//updates router iteration information and checks for breakpoints
```

Comment:
I like that you wrapped the router code up in a single routine.
Comment should say more precisely what router information (iteration count or net id being re-routed) is updated.  Should also say if this should be updated at the start of an iteration and before a net is re-routed, or after a net is re-routed.  (I.e. what is the convention?)

Reply:
resolved